csi-plug-n-play/sidecars/soda-csi-provisioner/external-provisioner/pkg/controller/controller.go

```diff
@@ -434,50 +459,32 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 	if err != nil {
 		klog.Fatalf("Error getting CSI driver name: %s", err)
 	}
-	klog.Infof("The Backend Driver Name is : %s ",backendDriverName)
-	klog.Infof("The provisioner.DriverName  is : %s ",p.driverName)
-
-	//TODO Make this code work with SODA API-Server (Issue is with Grpc version, soda uses v1.29.1 whereas csi-provisioner uses v1.26.0
-	/*client, err := opensds.GetClient("192.168.20.61:50040", "noauth")
-	if client == nil || err != nil {
-		klog.Errorf("get opensds client failed: %v", err)
-
-	}
+	klog.Infof("The Backend Driver Name and provisioner name is : %s , %s", backendDriverName, p.driverName)
 
-	if options.StorageClass.Provisioner == "soda-csi-block" {
+	if options.StorageClass.Provisioner == "soda-csi" {
 		for k, v := range options.StorageClass.Parameters {
-			klog.Infof("The parameters in the StorageClass are  : %s ===== %s",k,v)
+			klog.Infof("The parameters in the StorageClass are  : %s ===== %s", k, v)
 			if k == "profile" {
 
-				profile, errosds := client.GetProfile(v)
-				if errosds != nil {
-					klog.Infof("Got error in GetProfile  : %s ===== %s", errosds.Error())
-				}
-				klog.Infof("The profile name recieved in the storageClass is: %s ===== %s",profile.Name)
-				if backendDriverName != profile.Name {
-					return nil, controller.ProvisioningFinished, &controller.IgnoredError{
-						Reason: fmt.Sprintf("PVC doesnot match the current driver name : %s with expected %s",
-							p.driverName, profile.Name),
-					}
-				}
-			}
-		}
-	}*/
-	if options.StorageClass.Provisioner == "soda-csi-block" {
-		for k, v := range options.StorageClass.Parameters {
-			klog.Infof("The parameters in the StorageClass are  : %s ===== %s",k,v)
-			if k == "profile" {
-				if backendDriverName != v {
-					return nil, controller.ProvisioningFinished, &controller.IgnoredError{
-						Reason: fmt.Sprintf("PVC doesnot match the current driver name : %s with expected %s",
-							backendDriverName, v),
+				response, err := http.Get("http://soda-proxy:50029/getprofile/" + v)
+				if err != nil {
+					klog.Infof("Got error in GetProfile  : %s ===== %s", err.Error())
```

Comment:
Why at info level and not at error level

Reply:
@kumarashit Because this error will not cause any flow change, this error is expected when the soda-api-servers are not available, in this case the flow should just give a message and ignore the Volume Provisioning.

Anyways I will reframe the error message in a better way

Reply:
Ok. Got it